packages/sdks/web-component/src/lib/descope-wc/DescopeWc.ts

```diff
@@ -724,15 +724,19 @@ class DescopeWc extends BaseDescopeWc {
           return;
         }
 
+        this.#handleSdkResponse(sdkResp);
+
         this.logger.debug('polling - Got a response');
+
         if (sdkResp?.error) {
           this.logger.debug(
             'polling - Response has an error',
             JSON.stringify(sdkResp.error, null, 4),
           );
+
+          return;
```

Comment:
I think we should use the onError callback at this point instead of simply stopping the polling without a way to continue.

Reply:
But we need to stop the polling on error because otherwise we have an endless error loop (see the issue related to this PR)
Maybe in case of a specific error (mobile BG related error) we can continue polling

WDYT?

Reply:
I think this should be done regardless of native. If we stop polling - it should be considered a fatal error, because there's no way to recover at this point.